Libraries/libbom/Headers/bom/bom_format.h

```diff
@@ -30,6 +30,19 @@
 extern "C" {
 #endif
 
+#if _MSC_VER
+#define __BOM_PACKED_STRUCT_BEGIN __pragma(pack(push, 1))
+#define __BOM_PACKED_STRUCT_END __pragma(pack(pop))
+#else
+#define __BOM_PACKED_STRUCT_BEGIN
+#define __BOM_PACKED_STRUCT_END __attribute__((packed))
+#endif
+
+#if _MSC_VER
+__pragma(warning(push))
+__pragma(warning(disable: 4200))
```

Comment:
Hm, I thought that was C99 but not C++, although that may not be true with C++11? Will need to look.

Either way, if the warning disable is needed, I'd prefer to do it inline since libbom is used externally. (Although not on Windows AFAIK.)